### PR TITLE
Added EventBrite example

### PR DESCRIPTION
**Pull request by siowhoi**
Added EventBrite example and unit test
Screenshot 2023-12-01 at 3 36 36 pm

### CODE REVIEW

**Review by tnodell — APPROVED** [review]
Couple of small notes.

File: src/examples/eventbrite/package.json [hunk 1]
```diff
@@ -0,0 +1,61 @@
+{
+	"name": "@exogee/graphweaver-eventbrite",
+	"version": "0.2.6",
```
[the remark this answers is not in the document]
Comment (siowhoi) [reply src/examples/eventbrite/package.json]: fixed

File: src/examples/eventbrite/src/backend/client/index.ts [hunk 2]
```diff
@@ -0,0 +1,47 @@
+import { logger } from '@exogee/logger';
+import 'isomorphic-fetch';
+
+const eventbriteFetch = (token: string, path: string) => {
+	const url = new URL(path, 'https://www.eventbriteapi.com/');
+
+	const controller = new AbortController();
```
[the remark this answers is not in the document]
Comment (siowhoi) [reply src/examples/eventbrite/src/backend/client/index.ts:7]: It comes with node. https://blog.logrocket.com/complete-guide-abortcontroller-node-js/

File: src/examples/eventbrite/src/backend/schema/order/provider.ts [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+import { EventbriteOrder } from './entity';
+import { EventbriteOrderDataEntity } from './data-entity';
+import { eventbriteClient } from '../../client';
+
+export interface Context {
+	client: ReturnType<typeof eventbriteClient>;
+}
+
+export const createEventbriteOrderProvider = (token: string, organizationId: string) =>
```
[the remark this answers is not in the document]
Comment (siowhoi) [reply src/examples/eventbrite/src/backend/schema/order/provider.ts:11]: Fixed. Files are reorganised as suggested.

**Pull request merged**